dali_mqtt_daemon.py

```diff
@@ -191,11 +191,17 @@ def on_connect(
         MQTT_DALI2MQTT_STATUS.format(mqtt_base_topic), MQTT_AVAILABLE, retain=True
     )
     lamps = dali_scan(driver_object, max_lamps)
+    logger.info(
+        "Found %d lamps from excepted %d, scanned in range 0-%d",
```

Comment:
expected

Reply:
After #17 range is static 0-63 so I shortened this message to just "Found %d lamps"